extensions/amp-access/0.1/amp-access.js

```diff
@@ -125,6 +139,78 @@ export class AccessService {
   startInternal_() {
     actionServiceFor(this.win).installActionHandler(
         this.accessElement_, this.handleAction_.bind(this));
+
+    // Start authorization XHR immediately.
+    this.runAuthorization_();
+  }
+
+  /**
+   * @return {!Promise}
+   * @private
+   */
+  runAuthorization_() {
+    log.fine(TAG, 'Start authorization via ', this.config_.authorization);
+    this.toggleTopClass_('amp-access-loading', true);
+
+    // TODO(dvoytenko): produce READER_ID and create the URL substition for it.
+    return this.urlReplacements_.expand(this.config_.authorization)
+        .then(url => {
+          log.fine(TAG, 'Authorization URL: ', url);
+          return this.xhr_.fetchJson(url, {credentials: 'include'});
+        })
+        .then(response => {
+          log.fine(TAG, 'Authorization response: ', response);
+          this.toggleTopClass_('amp-access-loading', false);
+          onDocumentReady(this.win.document, () => {
+            this.applyAuthorization_(response);
+          });
+        })
+        .catch(error => {
+          log.error(TAG, 'Authorization failed: ', error);
+          this.toggleTopClass_('amp-access-loading', false);
+        });
+  }
+
+  /**
+   * @return {!JSONObjectDef} response
+   * @return
+   */
+  applyAuthorization_(response) {
+    const elements = this.win.document.querySelectorAll('[amp-access]');
+    for (let i = 0; i < elements.length; i++) {
+      this.applyAuthorizationToElement_(elements[i], response);
+    }
+  }
+
+  /**
+   * @param {!Element} element
+   * @return {!JSONObjectDef} response
+   * @return
+   */
+  applyAuthorizationToElement_(element, response) {
+    const expr = element.getAttribute('amp-access');
+    const on = evaluateAccessExpr(expr, response);
+
+    // TODO(dvoytenko): support templates
+
+    this.vsync_.mutate(() => {
+      if (on) {
+        element.removeAttribute('amp-access-off');
+      } else {
+        element.setAttribute('amp-access-off', '');
```

Comment:
No. The current state is that it's not on by default. However, another important aspect: authorization can run multiple times, so the code is repeatable.